start.sh

```diff
@@ -0,0 +1,10 @@
+#!bin/bash
+cd components/core;
+gnome-terminal -e "sudo python3 Main.py";
+cd ..;
+cd ..;
+cd ui/;
+gnome-terminal -e "sudo gulp serve";
```

Comment:
Why `gnome-terminal`?

Reply:
@iammosespaulr I think that using this line should take care of both the servers:
```
sudo python3 components/core/Main.py & cd ui/ & sudo gulp serve;
```